test/schedule_run_once_test.js

```diff
@@ -37,6 +37,32 @@ describe('Scheduled job with interval', ()=> {
       it('should run the job once', ()=> {
         assert.equal(count, 1);
       });
+    });
+
+    describe('after running job once', ()=> {
+
+      before(()=> {
+        TimeKeeper.travel(Date.now() + ms('30m'));
+      });
+
+      before(Ironium.runOnce);
+
+      it('should not run job again before next occurrence', ()=> {
```

Comment:
Well, that depends: if the interval is 1 hour, the job runs every hour on the hour. So if the time is 10:51, the next job is scheduled for 11:00, and if we move forward in time to 11:21 then it would in fact run again.